### ISSUE TITLE
Could the Network alpha paragraph be revised, please?

### ISSUE THREAD

**Issue by djigneo** [bmaltais/kohya_ss#1919]
I am attempting to understand https://github.com/bmaltais/kohya_ss/wiki/LoRA-training-parameters#network-alpha however it seems a bit unclear. Could this be clarified?

What's confusing is that the section seems to have several contradictory statements which makes it unclear what values make dampening stronger or weaker.

"The smaller the Network alpha value, the larger the stored LoRA neural net weights."
"For example, with an Alpha of 16 and a Rank of 32, the strength of the weight used is 16/32 = 0.5, meaning that the learning rate is only half as powerful as the Learning Rate setting."
These statements seem to contradict. If the strength of the weight is [Alpha]/[Rank], then smaller alpha values would mean _smaller_ stored values, right? I must be misunderstanding.

"If Network Alpha and Network Rank have the same value, the effect will be turned off."
"The default is 1 (that is, maximize the stored weight value)."
These statements seem to suggest that the dampening effect can be turned off by Network alpha values of [Network Rank] or 1. Does that mean it's impossible to specify an actual Network Alpha value of 1 (creating a dampening effect of 1/[Network Rank])?

"If the accuracy of LoRA after learning is not good enough, the weight data may be too small and collapsed to 0. In such a case, try lowering the Network Alpha value (=increasing the save weight value)."
This statement also seems to contradict the above statements. Wouldn't lowering the Network Alpha make the stored weights _smaller_, dampening the learning rate, rather than strengthening it?

I appreciate any clarity that could be added. Thanks!

**Comment by bmaltais** [comment]
Hello @djigneo I suggest opening a clarification question directly with kohya's sd-scripts repo. I only wrap his code with a GUI but he is the maserming behind the training it does: https://github.com/kohya-ss/sd-scripts

**Comment by djigneo** [comment]
Will do, appreciate it, thanks!
